mounts/org.osbuild.btrfs

```diff
@@ -27,11 +27,26 @@ SCHEMA_2 = """
   "options": {
     "type": "object",
     "additionalProperties": false,
-    "properties": {
-      "readonly": {
-        "description": "mount the source as a readonly device",
-        "type": "boolean"
-      }
+    "readonly": {
+      "description": "mount the source as a readonly device",
+      "type": "boolean"
+    },
+    "defaults": {
+      "description": "mount the source with the default option",
+      "type": "boolean"
+    },
+    "uid": {
+      "description": "mount the source with given uid",
+      "type": "int"
+    },
+    "gid": {
+      "description": "mount the source with given guid",
+      "type": "int"
+    },
+    "umask": {
+      "description": "mount the source with given umask",
+      "type": "string",
+      "pattern": "^[0-7][0-7][0-7]$"
```

Comment:
Are any of these valid for btrfs?  I checked the `mount(8)` and `btrfs(5)` man pages and it doesn't seem so.

Reply:
indeed! I cleaned that. Thanks!